Stop kolla fighting with kayobe over selinux state

jovial opened the pull request:
I found kolla-ansible was setting selinux policy to disabled so after host configure and reboot you always hit:

```
TASK [selinux : Abort SELinux configuration because reboot is disabled] ***************************************************************************************************************************************************************
Friday 01 December 2023  11:16:10 +0100 (0:00:01.314)       0:00:48.063 *******                                                                                                                                                        
fatal: [cpt07]: FAILED! => changed=false                                                                                                                                                                                               
  msg: |-                                                                                                                                                                                                                              
    SELinux state change requires a reboot, but selinux_do_reboot is false. Please run again with selinux_do_reboot set to true to reboot.                                                                                             
```
   
on the next host configure,


Change-Id: I0bfcb36224c403b04d04d51b4faad188a1dd2cac

markgoddard commented:
We should ensure these changes are in later branches before merging to ussuri. Do we need to cherry pick anything to victoria?

jovial committed:
Stop kolla fighting with kayobe over selinux state

I found kolla-ansible was setting selinux policy to disabled so after
host configure and reboot you always hit:

```
TASK [selinux : Abort SELinux configuration because reboot is disabled] ***************************************************************************************************************************************************************
Friday 01 December 2023  11:16:10 +0100 (0:00:01.314)       0:00:48.063 *******
fatal: [cpt07]: FAILED! => changed=false
  msg: |-
    SELinux state change requires a reboot, but selinux_do_reboot is false. Please run again with selinux_do_reboot set to true to reboot.
```

on the next host configure,

Change-Id: I0bfcb36224c403b04d04d51b4faad188a1dd2cac

jovial commented:
Have submitted upstream: https://review.opendev.org/c/openstack/kayobe/+/902369

markgoddard commented:
Please backport Pierre's patch instead

jovial commented:
done: https://github.com/stackhpc/kayobe/pull/211
